### PR TITLE
[4.0] Styling for FE editing buttons

### PR DESCRIPTION
**Pull request by C-Lodder**
Pull Request for Issue #18625

### Summary of Changes

Fix the styling for the `Save`, `Cancel` and `Versions` buttons in the frontend editor.

### CODE REVIEW

**Comment by ghost** [comment]
I have tested this item :red_circle: unsuccessfully on b0d9c29e0988154ed0a455f75f683b1165ace494<br /><br />Works but Icon at "Versions" needs Space on the right Side:
bildschirmfoto 2017-11-21 um 15 05 46

<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18775">issues.joomla.org/tracker/joomla-cms/18775</a>.</sub>

**Comment by C-Lodder** [comment]
@franz-wohlkoenig - This PR should have deleted the following file for you:

> templates/cassiopeia/html/layouts/joomla/form/field/contenthistory.php

If this file still exists for you, try manually deleting it.

**Comment by ghost** [comment]
delete templates/cassiopeia/html/layouts/joomla/form/field/contenthistory.php manually, Result is wrong Icon:

bildschirmfoto 2017-11-21 um 15 13 24

**Comment by C-Lodder** [comment]
Not the wrong icon, I purposely replaced the `folder` icon with the `code-fork` icon as it's a better visual indicator.

Same change applied to the toolbar in the backend template too ;)

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on b0d9c29e0988154ed0a455f75f683b1165ace494<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18775">issues.joomla.org/tracker/joomla-cms/18775</a>.</sub>

File: components/com_content/tmpl/form/edit.php [hunk 1]
```diff
@@ -147,21 +147,17 @@
...
-			</div>
-			<div class="btn-group">
-				<button type="button" class="btn" onclick="Joomla.submitbutton('article.cancel')">
-					<span class="icon-cancel"></span><?php echo JText::_('JCANCEL') ?>
-				</button>
-			</div>
+		<div class="mb-2">
+			<button type="button" class="btn btn-primary" onclick="Joomla.submitbutton('article.save')">
+				<span class="fa fa-check" aria-hidden="true"></span>
+				<?php echo JText::_('JSAVE') ?>
```
Comment (Quy) [inline components/com_content/tmpl/form/edit.php]: Add semicolon.

File: components/com_content/tmpl/form/edit.php [hunk 2]
```diff
@@ -147,21 +147,17 @@
...
+			</button>
+			<button type="button" class="btn btn-secondary" onclick="Joomla.submitbutton('article.cancel')">
+				<span class="fa fa-times-cancel" aria-hidden="true"></span>
+				<?php echo JText::_('JCANCEL') ?>
```
Comment (Quy) [inline components/com_content/tmpl/form/edit.php]: Add semicolon.

**Pull request merged**